### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by LILkillaBEE** [algolia/instantsearch-android-examples#1]
Hi,

I have same error for both projects:
```
Error:Execution failed for task ':ecommerce:transformClassesWithDexForDebug'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define Lorg/objectweb/asm/AnnotationVisitor;
```

```
Error:Execution failed for task ':media:transformClassesWithDexForDebug'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define Lorg/objectweb/asm/AnnotationVisitor;
```

What should I do to build it?

Thank you

**Comment by PLNech** [comment]
Hi @LILkillaBEE, thanks for your report!

I tried to reproduce your issue, but I was able to clone and run successfully. What version of Android Studio are you using?

Nonetheless, I just pushed the latest changes to this project. Could you try building them and let me know how it goes?

**Comment by LILkillaBEE** [comment]
Hey @PLNech ,

Latest master changes I could get before was at this commit:
https://github.com/algolia/instantsearch-android-examples/tree/1830f890cd862499587e11c016fcf98498719d1b

I'm using Android studio 2.2.3.
With your latest commits it's working now.

Thank you very much,
Nikola

**Comment by PLNech** [comment]
Hi Nikola, you're welcome! Glad to see you're building successfully. 

Let me know what you will make with InstantSearch Android, I will be happy to help you if I can!

**Comment by LILkillaBEE** [comment]
@PLNech issue is still there :(

There are two samples, media and ecommerce.
**Ecommerce** sample is building without errors, but **media** sample have the same error as before:
```
Error:Execution failed for task ':media:transformClassesWithDexForDebug'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define Lorg/objectweb/asm/AnnotationVisitor;

```

Please note that you will not get an error if you're just (re)building project in Android Studio, you have to build APK.

Thank you,
Nikola

**Comment by PLNech** [comment]
I found the root cause, the `media` example was out of sync with the latest versions of the main library! This explains why you had this build dependency issue only in that example app. 

I pushed [a fix](https://github.com/algolia/instantsearch-android-examples/commit/6a7ddfd763104c25915e9a7646416d159e62adba) to update the media example which also ensures this issue won't happen again. 
The media example now builds successfully, but let me know if you encounter any other issue 🙂
